builtin/providers/google/resource_pubsub_subscription.go

```diff
@@ -0,0 +1,81 @@
+package google
+
+import (
+	"fmt"
+	"google.golang.org/api/pubsub/v1"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourcePubsubSubscription() *schema.Resource {
+	return &schema.Resource{
+		Create: resourcePubsubSubscriptionCreate,
+		Read:   resourcePubsubSubscriptionRead,
+		Delete: resourcePubsubSubscriptionDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"topic": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"topic_computed": &schema.Schema{
```

Comment:
topic_computed is the canonical name of the topic that is required by the API for creating the subscription.  

however, it is already being computed from the name schema value.  I'll remove it shortly.